backend/src/flows.rs

```diff
@@ -78,14 +78,15 @@ pub struct FlowValue {
 #[derive(Deserialize, Serialize, Debug, Clone)]
 pub struct FlowModule {
     #[serde(default)]
-    pub input_transform: HashMap<String, InputTransform>,
+    #[serde(alias = "input_transform")]
```

Comment:
need to also change it in the openapi spec and the corresponding references in the frontend if we want to be exhaustive. But I can take care of that